src/abyssal_tome/app.py

```
@@ -0,0 +1,772 @@
+
+import clipman
+import flet as ft
+import flet_fastapi
+import regex as reg
+import requests
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport as GQL_Transport
+from starlette.middleware.cors import CORSMiddleware
+from tqdm.auto import tqdm
+from whoosh.fields import ID, TEXT, Schema
+from whoosh.index import create_in, open_dir
```

Flake8 (7.2.0)

[error] 'whoosh.index.open_dir' imported but unused

(F401)